docs/creating-claudie-backup/creating-claudie-backup.md

```diff
@@ -16,6 +16,109 @@ These are the only services that will have a PVC attached to it, the other are s
 
 ## Backing up Claudie
 
+### Manual backup
+
+Claudie is already deployed on an existing Management Cluster and at least 1 Input Manifest has been applied.
+
+Create a directory where the backup of the state will be stored.
+
+```bash
+mkdir claudie-backup
+```
+
+Put your Claudie inputmanifests into the created folder.
+
+We will now back up the state of the respective input manifests from MongoDB and MinIO.
+
+```bash
+kubectl get pods -n claudie
+
+NAME                                READY   STATUS      RESTARTS      AGE
+ansibler-6f4557cf74-b4dts           1/1     Running     0             18m
+builder-5d68987c86-qdfd5            1/1     Running     0             18m
+claudie-operator-6d9ddc7f8b-hv84c   1/1     Running     0             18m
+context-box-5d75bfffc6-d9qfm        1/1     Running     0             18m
+create-table-job-ghb9f              0/1     Completed   1             18m
+dynamodb-6d65df988-c626j            1/1     Running     0             18m
+kube-eleven-556cfdfd98-jq6hl        1/1     Running     0             18m
+kuber-7f8cd4cd89-6ds2w              1/1     Running     0             18m
+make-bucket-job-9mjft               0/1     Completed   0             18m
+minio-0                             1/1     Running     0             18m
+minio-1                             1/1     Running     0             18m
+minio-2                             1/1     Running     0             18m
+minio-3                             1/1     Running     0             18m
+mongodb-6ccb5f5dff-ptdw2            1/1     Running     0             18m
+scheduler-676bd4468f-6fjn8          1/1     Running     0             18m
+terraformer-66c6f67d98-pwr9t        1/1     Running     0             18m
+```
+
+To backup state from MongoDB execute the following command
+
+```bash
+kubectl exec -n claudie mongodb-6ccb5f5dff-ptdw2 -- sh -c 'mongoexport --uri=mongodb://$MONGO_INITDB_ROOT_USERNAME:$MONGO_INITDB_ROOT_PASSWORD@localhost:27017/claudie -c inputManifests --authenticationDatabase admin' > claudie-backup/inputManifests
```

Comment:
WDYT about replacing `mongodb-6ccb5f5dff-ptdw2` with `$YOUR-MONGODB-POD` / `<your-mongodb-pod>` or something like that in all occurrences?